component/Avatars/index.html

```diff
@@ -0,0 +1,175 @@
+            <h5 class="avatar-section-content">Small (4 rem)</h5>
+        </section>
+        <div class="iframe-container">
+            <code>
+                <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28255%2C222%2C197%2C1%29&t=nord&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Csection%2520class%253D%2522avatar-rd-txt-container%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522txt-rd%2520txt-rd-xl%2522%253E%253Ch3%253EVM%253C%252Fh3%253E%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522txt-rd%2520txt-rd-l%2522%253E%253Ch3%253EVM%253C%252Fh3%253E%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522txt-rd%2520txt-rd-m%2522%253E%253Ch3%253EVM%253C%252Fh3%253E%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522txt-rd%2520txt-rd-s%2522%253E%253Ch3%253EVM%253C%252Fh3%253E%253C%252Fdiv%253E%250A%253C%252Fsection%253E"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+            </code>
+        </div>
+
+        <!--  SQUARED TEXT AVATAR -->
+
+
+        <h4 class="avatar-subheading">> Squared Text Avatar</h4>
+        <h5 class="avatar-content">Squared Text Avatars shows the initals of your name in squared shape. As shown...</h5>
+        <section class="avatar-sq-txt-container">
+            <div class="txt-sq txt-sq-xl"><h3>VM</h3></div>
+            <div class="txt-sq txt-sq-l"><h3>VM</h3></div>
+            <div class="txt-sq txt-sq-m"><h3>VM</h3></div>
+            <div class="txt-sq txt-sq-s"><h3>VM</h3></div>
+        </section>
+        <section class="avatar-details">
+            <h5 class="avatar-section-content">Extra-Large (10 rem)</h5>
+            <h5 class="avatar-section-content">Large (8 rem)</h5>
+            <h5 class="avatar-section-content">Medium (6 rem)</h5>
+            <h5 class="avatar-section-content">Small (4 rem)</h5>
+        </section>
+        <div class="iframe-container">
+            <code>
+                <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28255%2C222%2C197%2C1%29&t=nord&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Csection%2520class%253D%2522avatar-sq-txt-container%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522txt-sq%2520txt-sq-xl%2522%253E%253Ch3%253EVM%253C%252Fh3%253E%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522txt-sq%2520txt-sq-l%2522%253E%253Ch3%253EVM%253C%252Fh3%253E%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522txt-sq%2520txt-sq-m%2522%253E%253Ch3%253EVM%253C%252Fh3%253E%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522txt-sq%2520txt-sq-s%2522%253E%253Ch3%253EVM%253C%252Fh3%253E%253C%252Fdiv%253E%250A%253C%252Fsection%253E"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+            </code>
+        </div>
+    </div>
+
+
+
+
+
+
```

Comment:
remove extra spacing.

Reply:
you can remove default margin in the website by ` html{ margin: 0; }` in your CSS.